src/Views/CategoryView.vala

```diff
@@ -130,23 +152,52 @@ public class AppCenter.CategoryView : Gtk.Stack {
                 child.destroy ();
             }
 
+            grid.add (recently_updated_box);
```

Comment:
Populate can be called more than once but `recently_updated_box` is never removed from the grid so a second attempt to add it would be an error.

Is it possible there are zero recently updated packages?  In that case this grid should be hidden.